amgut/lib/data_access/ag_data_access.py

```diff
@@ -272,10 +272,11 @@ def registerHandoutKit(self, ag_login_id, supplied_kit_id):
                             (ag_kit_id, barcode, sample_barcode_file)
                             VALUES (k_id, bc, bc || '.jpg');
                     END LOOP;
+                    DELETE FROM ag_handout_barcodes WHERE kit_id = %s;
                     DELETE FROM ag_handout_kits WHERE kit_id = %s;
                 END $do$;
                 """.format(ag_login_id, printresults)
-            TRN.add(sql, [supplied_kit_id] * 3)
+            TRN.add(sql, [supplied_kit_id] * 4)
```

Comment:
It changed from 3 to 4.  Any reason why?
And could you add that reason in a comment.  Otherwise, I think it is ready to merge.